### DISCUSSION TITLE
Security Policy

### DISCUSSION THREAD

**Discussion by rjm-smokey** [ngageoint/simple-features-geojson-java#8] [General]
Hello,

Is there a way the public is supposed to report security vulnerabilities for review? Most projects have one that informs researchers how to contact maintainers to submit vulnerabilities.

**Comment by bosborn** [comment]
You can [submit an issue](https://github.com/ngageoint/simple-features-geojson-java/issues/new/choose) and/or pull request.

If it is concerning jackson-databind, I usually bump the version when releasing a new version.  Still submit an issue if there is one though.

**Reply by bosborn** [reply]
Bumping `jackson-databind` to `2.17.0` [here](https://github.com/ngageoint/simple-features-geojson-java/commit/150a560a4881c17e4b04b00aeeb37cc7307b4100).

**Reply by bosborn** [reply]
Released in version [3.3.3](https://github.com/ngageoint/simple-features-geojson-java/releases/tag/3.3.3)

**Comment by rjm-smokey** [comment]
Cool, thanks!
